src/extensions/template_extension/TemplateFactory.cs

```diff
@@ -44,35 +45,44 @@ public static bool ListTemplates(string? templateFilter, string? languageFilter)
                 ConsoleHelpers.WriteLineError($"No matching templates found\n");
                 groups = GetTemplateGroups();
             }
-
-            var longNameLabel = "Name";
-            var shortNameLabel = "Short Name";
-            var languageLabel = "Language";
-
-            var widths = new int[3];
-            widths[0] = Math.Max(longNameLabel.Length, groups.Max(x => x.LongName.Length));
-            widths[1] = Math.Max(shortNameLabel.Length, groups.Max(x => x.ShortName.Length));
-            widths[2] = Math.Max(languageLabel.Length, groups.Max(x => x.Languages.Length));
-
-            var hideLongName = !Console.IsOutputRedirected && Screen.GetRightColumn() < widths.Sum() + 4 * 2 + 1;
-
-            if (!hideLongName) Console.Write($"{longNameLabel.PadRight(widths[0])}    ");
-            Console.WriteLine($"{shortNameLabel.PadRight(widths[1])}    {languageLabel.PadRight(widths[2])}");
-
-            if (!hideLongName) Console.Write($"{"-".PadRight(widths[0], '-')}    ");
-            Console.WriteLine($"{"-".PadRight(widths[1], '-')}    {"-".PadRight(widths[2], '-')}");
-
-            for (int i = 0; i < groups.Count; i++)
-            {
-                var longName = groups[i].LongName;
-                var shortName = groups[i].ShortName.Replace('_', '-');
-                var languages = groups[i].Languages;
-
-                if (!hideLongName) Console.Write($"{longName.PadRight(widths[0])}    ");
-                Console.WriteLine($"{shortName.PadRight(widths[1])}    {languages.PadRight(widths[2])}");
+            if (InOutPipeServer.IsInOutPipeServer)
```

Comment:
If you don't indent all this and have it under an `else` you won't have a merge conflict with the changes I just made.

Reply:
also ... it'll be more similar to the other places we check the `IsInOutPipeServer` property...

Reply:
fixed